applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -1932,6 +1876,37 @@ Vector UPwSmallStrainElement<TDim, TNumNodes>::GetPressureSolutionVector()
     return result;
 }
 
+template <unsigned int TDim, unsigned int TNumNodes>
+void UPwSmallStrainElement<TDim, TNumNodes>::CalculateAnyOfMaterialResponse(
```

Comment:
For the naming of this function, it computes the indicated things for all i.p.
In other cases we indicated that with a trailing s. That can be done here too: CalculateMaterialResponses

Reply:
Agreed, either this, or we rename to something like `CalculateAnyOfStressesStrainsOrConstitutiveMatrices` (see other comment)